i18n/bundle/bundle.go

```diff
@@ -262,3 +263,18 @@ func isNumber(n interface{}) bool {
 	}
 	return false
 }
+
+func toMap(input interface{}) map[string]interface{} {
+	v := reflect.ValueOf(input)
+	switch v.Kind() {
+	case reflect.Map:
+		data, _ := input.(map[string]interface{})
+		return data
+	case reflect.Ptr:
+		return toMap(v.Elem().Interface())
+	case reflect.Struct:
+		return structToMap(v)
+	default:
+		panic(fmt.Errorf("i18n: cannot handle type %s", v.Kind()))
```

Comment:
As a library, I would prefer not to panic under any circumstances. We should just silently discard the data if it isn't a type we expect.